pkg/components/ebpf/common/http2grpc_transform.go

```diff
@@ -346,7 +358,7 @@ func http2FromBuffers(parseContext *EBPFParseContext, event *BPFHTTP2Info) (requ
 			method, path, contentType, ok := readMetaFrame(parseContext, connID, framer, ff)
 
 			if path == "" {
-				path = "."
+				path = "*"
```

Comment:
@mariomac  I believe this was accidentally changed when reverting the Paths from Glob to Regex. The Path here is the route, unknown, which we mark with asterisk.